.travis.yml

```diff
@@ -7,3 +7,19 @@ ghc:
   - "8.2"
   - "8.0"
   - "7.10"
+
+before_deploy: |
+  if [[ $TRAVIS_HASKELL_VERSION == "8.6.5" ]]; then
+    tar -C dist/build/dotenv -cvf dotenv-linux-x86_64-static.tar.gz dotenv
+    cabal install -f 'static'
+  fi
+
+deploy:
+  provider: releases
+  api_key: "$GITHUB_TRAVIS_DEPLOY"
+  file: "dotenv-linux-x86_64-static.tar.gz"
+  skip_cleanup: true
+  draft: true
```

Comment:
@CristhianMotoche draft releases need to be manually approved, any particular reason why we want to add an extra step here?

Reply:
Hey @sestrella I think with the draft release, we're able to manually add release notes (e.g. the latest changes), test that the binary works correctly before doing a real release.